### DISCUSSION TITLE
copilot payment done but not activating.

### DISCUSSION THREAD

**Discussion by edugrip** [community/community#32634] [Copilot Conversations]
Recently I made a payment for github copilot. Still it has stopped giving suggestion and not activating again. 
The message on my profile says that "Extended trial period has ended"

**Comment by samuel-ma** [comment]
@edugrip 
Are you sure the payment was successful
Did Github send a message to verify your payment on your email
Please verify that the payment was successful first

**Comment by davecheney** [comment]
@edugrip i'm sorry you are having difficulties. Looking at your telemetry it looks like you are making and receiving completions. Can you please tell me a bit more about what you tried, and what happened. Thank you.
